sdk/messaging/src/main/java/com/klaviyo/messaging/KlaviyoWebView.kt

```diff
@@ -130,9 +136,26 @@ class KlaviyoWebView : WebViewClient(), WebViewCompat.WebMessageListener {
                 is KlaviyoWebFormMessageType.AggregateEventTracked -> {
                     Registry.get<ApiClient>().enqueueAggregateEvent(messageType.payload)
                 }
+                is KlaviyoWebFormMessageType.DeepLink -> {
```

Comment:
nit: If its more than just a function call, maybe a when case should be pulled into a separate function for readability.